yesno.d.ts

```diff
@@ -0,0 +1,16 @@
+declare namespace yesno {
+  export interface InvalidHandlerOptions {
```

Comment:
More accurately it is `export type InvalidHandlerOptions = Omit<Options, 'invalid'>`, but built-in Omit type is since TypeScript 3.5 .